discovery-provider/es-indexer/src/indexers/BaseIndexer.ts

```diff
@@ -31,7 +31,7 @@ export abstract class BaseIndexer<RowType> {
   }
 
   baseSelect() {
-    return `select * from ${this.tableName} where is_current `
```

Comment:
this will still work actually, it's just always true

Reply:
yeah the query planner can't know that tho... want to remove any chance it resorts to scanning